data/transactions/test_files.txt

```diff
@@ -5,7 +5,6 @@ commit.json
 count.json
 delete.json
 error-labels.json
-errors-client.json
```

Comment:
This was missing from the test_files.txt list before, so we weren't running it. After it was added, it revealed that this test file was failing. I think the file was prematurely synced. It will be synced as part of https://jira.mongodb.org/browse/CXX-1799, which requires implementing the `assertSessionTransactionState` test operation. (It isn't done yet for libmongoc's test runner either).

Reply:
Nice detective work!